### DISCUSSION TITLE
RTX Super Resolution support?

### DISCUSSION THREAD

**Discussion by jedthe3rd** [jellyfin/jellyfin-desktop#389] [Ideas]
I am curious if this would be possible to support.

https://nvidia.custhelp.com/app/answers/detail/a_id/5448/~/rtx-video-super-resolution-faq

Now that they have enabled it. Nice to have it with Nvidia Shield but would be cool to get it on the desktop.

-James

**Comment by iwalton3** [comment]
There is something like this in the MPV Shim client: https://github.com/iwalton3/default-shader-pack

I haven't ported the UI/shader setting system but you can manually install shaders using the MPV config.

**Reply by EnterOFF** [reply]
It would be really nice to have this feature in the desktop version.

**Comment by Medic31w3** [comment]
The latest version of MVP has support for RTX Super Resolution. https://github.com/mpv-player/mpv/releases/tag/v0.39.0

Would love to know if the Media Player has been updated with this new version and potentially make use of this feature without having to resort to the Shim.....

Thoughts?

**Comment by warmach1ne** [comment]
is there any update regarding the RTX SR with Jellyfin Desktop?
